src/Util/Root.cpp

```diff
@@ -26,22 +26,21 @@ void Root::Update() {
     struct StackInfo {
         std::shared_ptr<GameObject> m_GameObject;
         Transform m_ParentTransform;
-        bool operator()(const StackInfo& lhs, const StackInfo& rhs) const {
-            return  lhs.m_GameObject->GetZIndex()> rhs.m_GameObject->GetZIndex();
-        }
     };
-    auto compareFunction = [](const StackInfo& a, const StackInfo& b) {
-        return a.m_GameObject->GetZIndex() > b.m_GameObject->GetZIndex();
-    };
-    
+
     std::vector<StackInfo> stack;
     stack.reserve(m_Children.size());
 
     for (const auto &child : m_Children) {
         stack.push_back(StackInfo{child, Transform{}});
     }
 
-    std::priority_queue<StackInfo, std::vector<StackInfo>, decltype(compareFunction)> renderQueue(compareFunction);
+    auto compareFunction = [](const StackInfo &a, const StackInfo &b) {
+        return a.m_GameObject->GetZIndex() > b.m_GameObject->GetZIndex();
+    };
+    std::priority_queue<StackInfo, std::vector<StackInfo>,
+                        decltype(compareFunction)>
+        renderQueue(compareFunction);
```

Comment:
why this format looks so ugly  ![sadge](https://cdn.7tv.app/emote/61630205c1ff9a17cc396522/1x.webp)